### PR TITLE
chore: generate documentation for cli

### PR DESCRIPTION
**Pull request by drpebcak**

### CODE REVIEW

**Comment by ibuildthecloud** [comment]
Oh no, now you'll see how terrible the descriptions are (or lack thereof) on the CLI stuff.

File: tools/gendocs/main.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+---
+`
+
+func main() {
+	cmd := gptscript.New()
+	cmd.DisableAutoGenTag = true
+
+	files, err := filepath.Glob("docs/docs/100-reference/01-command-line/gptscript_*.md")
+	if err != nil {
+		log.Fatal(err)
```
Comment (njhale) [inline tools/gendocs/main.go:26]: looks like the import for `log` is missing.

**Review by njhale — CHANGES_REQUESTED** [review]

File: tools/gendocs/main.go (hunk 1 above)
Comment (drpebcak) [reply tools/gendocs/main.go:26]: hmm? it's up on line 5

Comment (njhale) [reply tools/gendocs/main.go:26]: Ah, yup. Should stop reviewing from my phone!

**Review by njhale — APPROVED** [review]

**Review by g-linville — APPROVED** [review]

**Review by tylerslaton — APPROVED** [review]

**Pull request merged**